meshtastic/interfaces/ble/receive_service.py

```
@@ -33,82 +34,182 @@
+        else:
+            raw_is_closing = getattr(state_manager, "is_closing", None)
+            state_is_closing: bool | None = None
+            if callable(raw_is_closing) and not _is_unconfigured_mock_callable(
+                raw_is_closing
+            ):
+                try:
+                    result = raw_is_closing()
+                    if isinstance(result, bool):
+                        state_is_closing = result
+                except (
+                    Exception
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)